src/components/AudioCell.vue

```diff
@@ -129,80 +140,90 @@ const isInitializingSpeaker = computed(
 );
 const audioItem = computed(() => store.state.audioItems[props.audioKey]);
 
+const activeAudioKey = computed(() => store.getters.ACTIVE_AUDIO_KEY);
+
 const uiLocked = computed(() => store.getters.UI_LOCKED);
 
 const isMultiSelectEnabled = computed(
   () => store.state.experimentalSetting.enableMultiSelect
 );
 
-const setActiveAudioKey = () => {
-  if (store.getters.ACTIVE_AUDIO_KEY !== props.audioKey) {
-    store.dispatch("SET_ACTIVE_AUDIO_KEY", { audioKey: props.audioKey });
-    store.dispatch("SET_SELECTED_AUDIO_KEYS", { audioKeys: [props.audioKey] });
-  }
+const selectAndSetActiveAudioKey = () => {
+  store.dispatch("SET_ACTIVE_AUDIO_KEY", { audioKey: props.audioKey });
+  store.dispatch("SET_SELECTED_AUDIO_KEYS", { audioKeys: [props.audioKey] });
 };
+// 複数選択のクリック判定
```

Comment:
めちゃ細かいですけどこれは判定するかどうかを決める関数ぽくなっちゃうので、こうですかね
```suggestion
// 複数選択のクリック時
```